{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "cveId": "CVE-2025-27786",
    "assignerOrgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
    "state": "PUBLISHED",
    "assignerShortName": "GitHub_M",
    "dateReserved": "2025-03-06T18:06:54.461Z",
    "datePublished": "2025-03-19T20:37:35.307Z",
    "dateUpdated": "2025-03-19T20:37:35.307Z"
  },
  "containers": {
    "cna": {
      "title": "Applio allows arbitrary file removal in core.py",
      "problemTypes": [
        {
          "descriptions": [
            {
              "cweId": "CWE-22",
              "lang": "en",
              "description": "CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal')",
              "type": "CWE"
            }
          ]
        }
      ],
      "metrics": [
        {
          "cvssV4_0": {
            "attackVector": "NETWORK",
            "attackComplexity": "LOW",
            "attackRequirements": "NONE",
            "privilegesRequired": "NONE",
            "userInteraction": "NONE",
            "vulnConfidentialityImpact": "NONE",
            "vulnIntegrityImpact": "HIGH",
            "vulnAvailabilityImpact": "HIGH",
            "subConfidentialityImpact": "NONE",
            "subIntegrityImpact": "NONE",
            "subAvailabilityImpact": "NONE",
            "baseScore": 7.8,
            "baseSeverity": "HIGH",
            "vectorString": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:N/VI:H/VA:H/SC:N/SI:N/SA:N/E:P",
            "version": "4.0"
          }
        }
      ],
      "references": [
        {
          "name": "https://securitylab.github.com/advisories/GHSL-2024-341_GHSL-2024-353_Applio/",
          "tags": [
            "x_refsource_CONFIRM"
          ],
          "url": "https://securitylab.github.com/advisories/GHSL-2024-341_GHSL-2024-353_Applio/"
        },
        {
          "name": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/core.py#L329",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/core.py#L329"
        },
        {
          "name": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/tts/tts.py#L133",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/tts/tts.py#L133"
        }
      ],
      "affected": [
        {
          "vendor": "IAHispano",
          "product": "Applio",
          "versions": [
            {
              "version": "<= 3.2.8-bugfix",
              "status": "affected"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
        "shortName": "GitHub_M",
        "dateUpdated": "2025-03-19T20:37:35.307Z"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "Applio is a voice conversion tool. Versions 3.2.8-bugfix and prior are vulnerable to arbitrary file removal in core.py. `output_tts_path` in tts.py takes arbitrary user input and passes it to `run_tts_script` function in core.py, which checks if the path in `output_tts_path` exists, and if yes, removes that path, which leads to arbitrary file removal. As of time of publication, no known patches are available."
        }
      ],
      "source": {
        "advisory": "GHSA-4vf3-c78h-49g8",
        "discovery": "UNKNOWN"
      }
    }
  }
}